paparazzi/paparazzi/src/test/java/app/cash/paparazzi/internal/resources/CommentTrackingXmlPullParserTest.kt

```diff
@@ -0,0 +1,226 @@
+package app.cash.paparazzi.internal.resources
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import java.io.StringReader
+
+class CommentTrackingXmlPullParserTest {
+  @Test
+  fun test() {
+    StringReader(SIMPLE_RES).use {
+      val parser = CommentTrackingXmlPullParser().apply {
+        setInput(it)
+      }
+
+      // <resources>
+      parser.nextToken()
```

Comment:
similar to question in ValueResourceXmlParser, is this needed?

Reply:
ditto